jdaviz/app.py

```diff
@@ -1655,6 +1655,17 @@ def _on_new_viewer(self, msg, vid=None, name=None):
             msg.cls, data=msg.data, show=False)
         viewer.figure_widget.layout.height = '100%'
 
+        links_control_plugin = self._jdaviz_helper.plugins.get('Links Control', None)
+        if links_control_plugin is not None:
+            viewer.state.linked_by_wcs = links_control_plugin.link_type.selected == 'WCS'
+        elif len(self._viewer_store):
+            # The plugin would only not exist for instances of Imviz where the user has
+            # intentionally removed the Links Control plugin, but in that case we will
+            # adopt "linked_by_wcs" from the first (assuming all are the same)
+            # NOTE: deleting the default viewer is forbidden both by API and UI, but if
+            # for some reason that was the case here, linked_by_wcs will default to False
+            viewer.state.linked_by_wcs = list(self._viewer_store.values()[0]).state.linked_by_wcs
```

Comment:
Did you close the parenthesis at the wrong place?
```suggestion
            viewer.state.linked_by_wcs = list(self._viewer_store.values())[0].state.linked_by_wcs
```

Reply:
woops!  Good catch!